static/styles/portico.scss

```diff
@@ -2148,6 +2148,19 @@ input.new-organization-button {
 #devtools-page {
     max-width: 800px;
     margin: 0 auto;
+    padding-top: 100px;
```

Comment:
I don't think padding-top is the right way to fix this; I suspect more likely the `margin 0 auto` or something is the bug here.